doc/radosgw/cloud-transition.rst

```diff
@@ -75,84 +83,88 @@ Contains a list of ``acl_mappings``.
 * ``acl_mapping`` (container)
 
 Each ``acl_mapping`` structure contains ``type``, ``source_id``, and ``dest_id``. These
-will define the ACL mutation that will be done on each object. An ACL mutation allows converting source
-user id to a destination id.
+define the ACL mutation to be done on each object. An ACL mutation allows converting source
+userid to a destination userid.
 
 * ``type`` (id | email | uri)
 
-ACL type: ``id`` defines user id, ``email`` defines user by email, and ``uri`` defines user by ``uri`` (group).
+ACL type: ``id`` defines userid, ``email`` defines user by email,
+and ``uri`` defines user by ``uri`` (group).
 
 * ``source_id`` (string)
 
 ID of user in the source zone.
 
 * ``dest_id`` (string)
 
-ID of user in the destination.
+ID of user on the destination.
 
 * ``target_path`` (string)
 
-A string that defines how the target path is created. The target path specifies a prefix to which
-the source 'bucket-name/object-name' is appended. If not specified the target_path created is "rgwx-${zonegroup}-${storage-class}-cloud-bucket".
+A string that defines how the target path is constructed. The target path
+specifies a prefix to which the source bucket-name/object-name is appended.
+If not specified the ``target_path`` ``rgwx-${zonegroup}-${storage-class}-cloud-bucket``.
 
 For example: ``target_path = rgwx-archive-${zonegroup}/``
 
 * ``target_storage_class`` (string)
 
-A string that defines the target storage class to which the object transitions to. If not specified, object is transitioned to STANDARD storage class.
+A string that defines the target storage class to which the object transitions.
+If not specified, the object is transitioned to the ``STANDARD`` storage class.
 
 * ``retain_head_object`` (true | false)
 
-If true, retains the metadata of the object transitioned to cloud. If false (default), the object is deleted post transition.
-This option is ignored for current versioned objects. For more details, refer to section "Versioned Objects" below.
+If ``true``, retains the metadata of the object transitioned to the cloud service.
+If ``false`` (default), the object is deleted after the transition.
+This option is ignored for current-versioned objects. For more details,
+refer tthe "Versioned Objects" section below.
 
 
 S3 Specific Configurables:
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Currently cloud transition will only work with backends that are compatible with AWS S3. There are
-a few configurables that can be used to tweak its behavior when accessing these cloud services:
-
-::
+Currently cloud transition will work only with backends that are compatible with
```

Comment:
s/Currently/Currently,/

Reply:
(Although an adverb of time like "Currently" has bitrot built right into it when it occurs in the context of a suite of documentation like this one.)